core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.ts

```diff
@@ -42,22 +50,29 @@ export class ExplorationFooterComponent {
   resizeSubscription!: Subscription;
   contributorNames: string[] = [];
   hintsAndSolutionsAreSupported: boolean = true;
+  numberofCheckpoints: number = 0;
+  checkpointArray: number[] = [0];
+  expInfo: LearnerExplorationSummaryBackendDict;
+  completedWidth: number = 0;
```

Comment:
Unsure what this means..

Reply:
Could you please explain how this was resolved?

Reply:
I thought the objective was to explain the purpose of the variables. Am I supposed to do something else?